### ISSUE TITLE
CI builds continuously spawn new instances

### ISSUE THREAD

**Issue by nvella** [OpenLiveWriter/OpenLiveWriter#838]
Builds from AppVeyor continuously spawn new instances of themselves. Renaming Update.exe and starting OpenLiveWriter.exe directly seems workaround the issue, however this is a considerable roadblock for many users that wish to try out dev builds. 

OpenLiveWriter.ApplicationMain.LaunchAdditionalInstance seems to be called each time an extra instance is spawned. Below is a stack trace of one of these calls;

```
at OpenLiveWriter.ApplicationMain.LaunchAdditionalInstance(String[] args)
   at OpenLiveWriter.ApplicationMain.LaunchAction(String[] args, Boolean isFirstInstance)
   at OpenLiveWriter.CoreServices.SingleInstanceApplicationManager.LaunchActionThreadWithState.ThreadProc()
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
```

## Workarounds
Numerous workarounds have been outlined in issue #786 for specifically installing the #810 fix. Installing a recent dev build would involve applying these instructions with a more recent build artifact rather than the build for #810.
- http://kgbthatsme.blogspot.com/2019/05/open-live-writer-works-again-with.html
- http://www.thanislim.com/2019/03/how-to-fix-open-live-writer-error-400.html

In summary:
1. Rename `%localappdata%\OpenLiveWriter\Update.exe` to another file name.
2. Change all Open Live Writer shortcuts to point towards `%localappdata%\OpenLiveWriter\app-x.y.z\OpenLiveWriter.exe` where x.y.z is the installed app version.

Please keep in mind that these workarounds break auto-updating.

**Comment by nvella** [comment]
Other interesting behavior - a `app-0.6.2` folder briefly appears in `%localappdata%\OpenLiveWriter` before disappearing when the new instance is launched.

**Comment by JoniWebb** [comment]
OK  never mind!  I fixed it.  It was a mistake on my end.  Whew!!!  I get so nervous everytime I can't publish.  
Thanks!J


-----Original Message-----
